### DISCUSSION TITLE
GEN24_Ladesteuerung v0.12.1

### DISCUSSION THREAD

**Discussion by wiggal** [wiggal/GEN24_Ladesteuerung#44] [Announcements]
Änderungen in html/7_tab_Diagram.php und html/funktion_Diagram.php
- Balkendiagramme, Optionsauswahl usw. hinzugefügt.

<hr /><em>This discussion was created from the release <a href='https://github.com/wiggal/GEN24_Ladesteuerung/releases/tag/v0.12.1'>GEN24_Ladesteuerung v0.12.1</a>.</em>

**Comment by modezz** [comment]
Eben gepullt & folgendes noch in crontab gepackt fürs 24h logging

`*/3 * * * * /home/pi/GEN24_Ladesteuerung/start_PythonScript.sh SymoGen24Controller2.py logging`

wieder TOP - danke!
SG

**Reply by wiggal** [reply]
da der GEN24 die Daten der verwendeten Zählerstände nur alle 5 Minuten aktualisiert, habe ich */5 * * * * eingetragen. Häufiger bringt eigentlich nichts.

Blos zur Info
schönen Gruß
